operator/tasks/allocator_aws/allocator_aws.go

```diff
@@ -12,17 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package allocatoraws
```

Comment:
I'd suggest to make the package name consistent with the directory name here, i.e. either rename the directory to `allocatoraws` or the package to `allocator_aws`.

Reply:
That has changed alraead.